awx/sso/fields.py

```diff
@@ -214,7 +225,7 @@ def __init__(self, **kwargs):
         super(LDAPDNField, self).__init__(**kwargs)
         self.validators.append(validate_ldap_dn)
 
-    def run_validation(self, data=empty):
+    def run_validation(self, data=fields.empty):
```

Comment:
I'm not sure how far you want to go in this PR, but `empty` is a second-order import and comes from DRF. Might be safer to import it directly from DRF into this module. Also, `empty` doesn't seem to be referenced in `awx.conf.fields`.

Reply:
Yea, I was just looking at this one.  I want to stare at it some more.